tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php

```diff
@@ -288,7 +296,7 @@ public function testPostNewThatWillBeTagged()
         $tags = $entry->getTags();
 
         $this->assertCount(2, $tags);
-        $this->assertEquals('wallabag', $tags[0]->getLabel());
+        $this->assertContains('wallabag', $tags);
```

Comment:
Oh sorry didn't see we were checking the first item, I though we were only checking the label.
It's ok.